client/js/components/GoogleMapSearch/GoogleMapsSearch.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from "react";
+import { View } from "react-native";
+import { GooglePlacesAutocomplete } from "react-native-google-places-autocomplete";
+import styles from "./styles";
+import globalStyles from "../../config/globalStyles";
+
+class GooglePlacesInput extends Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    return (
+      <View style={styles.container}>
+        <GooglePlacesAutocomplete
+          placeholder="Where are you going?"
+          minLength={2}
+          autoFocus={false}
+          returnKeyType={"search"}
+          keyboardAppearance={"light"}
+          listViewDisplayed="auto"
+          fetchDetails={true}
+          renderDescription={row => row.description}
+          onPress={(data, details = null) => {
+            this.props.updateContextLocation(details.geometry.location);
+            console.log(details);
+          }}
+          getDefaultValue={() => ""}
+          query={{
+            key: "AIzaSyDT98OenVboMDlCOGFHCzMI7l0W5Y7-9VM",
+            language: "en"
+          }}
+          styles={{
```

Comment:
Put the styles into styles.js.

Reply:
This is a default prop. I can't transfer this object to the styles.js. I told sid and he said it was fine.